src/main/resources/application.yml

```diff
@@ -61,12 +61,16 @@ api:
           rejected-document-welsh: ${TEMPLATE_PROXY_REJECTED_DOCUMENT_EMAIL_WELSH}
           rejected-signature-english: ${TEMPLATE_PROXY_REJECTED_SIGNATURE_EMAIL_ENGLISH}
           rejected-signature-welsh: ${TEMPLATE_PROXY_REJECTED_SIGNATURE_EMAIL_WELSH}
+          nino-not-matched-english: ${TEMPLATE_PROXY_NINO_NOT_MATCHED_EMAIL_ENGLISH}
+          nino-not-matched-welsh: ${TEMPLATE_POSTAL_NINO_NOT_MATCHED_EMAIL_WELSH}
```

Comment:
template says POSTAL, should it be PROXY

Reply:
noticed after pushing, updated now